plugins/modules/ec2.py

```diff
@@ -577,6 +577,297 @@
 
 '''
 
+RETURN = r'''
+changed:
+    description: If the EC2 instance has changed.
+    type: bool
+    returned: always
+    sample: true
+instances:
+    description: The instances.
+    type: list
+    returned: always
+    contains:
+        ami_launch_index:
+            description: The AMI launch index, which can be used to find this instance in the launch group.
+            type: int
+            returned: always
+            sample: 0
+        architecture:
+            description: The architecture of the image.
+            type: str
+            returned: always
+            sample: "x86_64"
+        block_device_mapping:
+            description: Any block device mapping entries for the instance.
+            type: dict
+            returned: always
+            sample: {
+                "/dev/xvda": {
+                    "delete_on_termination": true,
+                    "status": "attached",
+                    "volume_id": "vol-06d364586f5550b62"
+                }
+            }
+        dns_name:
+            description: If the EC2 instance has changed.
+            type: str
+            returned: always
+            sample: ""
```

Comment:
```suggestion
            sample: "ec2-203-0-113-1.z-2.compute-1.amazonaws.com"
```
203.0.113.0/24 is a reserved block that can safely be used in documentation.  The same can be used for public_dns_name below.

Reply:
@jillr Thank you. Just added it (even for public_dns_name).